src/test/compile-fail/variant-namespacing.rs

```diff
@@ -1,49 +0,0 @@
-// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
-// file at the top-level directory of this distribution and at
-// http://rust-lang.org/COPYRIGHT.
-//
-// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
-// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
-// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
-// option. This file may not be copied, modified, or distributed
-// except according to those terms.
-
-// aux-build:variant-namespacing.rs
-
-extern crate variant_namespacing;
-pub use variant_namespacing::XE::*;
```

Comment:
This test ensures that variants are defined in both namespaces, it shouldn't be deleted, it should be tweaked to avoid glob imports instead.
`pub use variant_namespacing::XE::*` -> `pub use variant_namespacing::XE::{XStruct, XTuple, XUnit}` etc.